esmvalcore/config-developer.yml

```
@@ -203,7 +208,11 @@ obs4mips:
   input_dir:
     default: 'Tier{tier}/{dataset}'
     RCAST: '/'
-  input_file: '{short_name}_{dataset}_{level}_{version}_*.nc'
+    IPSL: '{realm}/{short_name}/{freq}/{grid}/{institute}/${dataset}/{latest_version}'
```

```suggestion
    IPSL: '{realm}/{short_name}/{freq}/{grid}/{institute}/{dataset}/{latest_version}'
```
is the `$` in the filename intended?

---

NO. Fixed